src/Sulu/Bundle/AdminBundle/Resources/js/containers/Application/colors.scss

```diff
@@ -8,13 +8,12 @@ $alto: #ddd;
 $black: #000;
 $blue: #00f;
 $blueZodiac: #112a46;
-$cloudBurst: #1c2945;
+$cloudBurst: #29354f;
```

Comment:
we used http://chir.ag/projects/name-that-color/ for naming our colors. would do that here too and use `pickledBluewood` for this one 🙂